model/project.go

```diff
@@ -1294,6 +1294,19 @@ func (p *Project) FindBuildVariant(build string) *BuildVariant {
 	return nil
 }
 
+func (p *Project) FindMatchingBuildVariants(bvRegex *regexp.Regexp) []string {
+	var res []string
+	if bvRegex.String() == "all" || bvRegex.String() == "" {
```

Comment:
It seems kind of weird to match "all" or "" here and treat them as actual build variant names when they're not real build variant names. Should `ResolvePatchVTs` instead check for the special case "all"/"" string before passing the regexp to `FindMatchingBuildVariants`? "all"/"" get resolved into all the real build variants names in `ResolvePatchVTs` anyways.

Reply:
done